woocommerce/woocommerce-gutenberg-products-block
# Remove page view tracking in favor of jetpack

This PR removes all page view tracking since we moved them to Jetpack.

## Testing Instructions

1. On a live, tracked website, running Jetpack, visit Cart/Checkout blocks.
2. Open [Tracks Vigilante](https://github.com/Automattic/tracks-chrome-extension)
3. You should not see `woocommerceanalytics_order_confirmation_view`, `woocommerceanalytics_cart_view`, `woocommerceanalytics_checkout_view`.
4. If you're running this PR https://github.com/Automattic/jetpack/pull/34421 from Jetpack, you should only see one instance of that event, not 2.



## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:
